src/configure/configure.ts

```diff
@@ -379,6 +389,7 @@ class Orchestrator {
     }
 
     private setDefaultRepositoryDetails(): void {
+        this.isLocalRepo = true;
```

Comment:
This can be defined as extension variable so that can be used in other files/classes as well?

Reply:
I don't think we are using this as of now, but yes, since can be used in future, will update.